descriptors/ModuleDescriptor-template.json

```diff
@@ -76,7 +76,7 @@
     },
     {
       "id": "source-storage-records",
-      "version": "1.0"
+      "version": "3.0"
```

Comment:
mod-source-storage-records has not increased source-storage-records interface version. Please leave it unchanged

Reply:
It has increased https://github.com/folio-org/mod-source-record-storage/releases/tag/v3.0.0. So why not update it?

Reply:
3.0.0 is a module version not interface